metacatalog_api/templates/license.html

```diff
@@ -38,4 +38,8 @@
         </div>
     </div>
 </div>
-<script src="https://cdn.tailwindcss.com"></script>
\ No newline at end of file
+
+{% set embedded = request.query_params.get('embedded') == 'true' %}
+{% if not embedded %}
+    <script src="https://cdn.tailwindcss.com"></script>
+{% endif %}
```

Comment:
I don't have build tools installed as all html templates are static.

Is there a way to link a pre-build minified tailwind, without building it for the project? Otherwise I would ignore this comment.